boudicca.base/publisher-event-html/src/main/resources/static/js/main.js

```diff
@@ -10,6 +10,64 @@ document.addEventListener("DOMContentLoaded", () => {
   const categorySelect = document.getElementById("category");
   const categoryFieldSets = document.querySelectorAll("[data-category-wanted]");
 
+  ////// TODO: find a way to add event listener to newly added events after "merh laden" button is pressed
+  ////// TODO: refactor prototype code, follow existing code style
+  const events = document.querySelectorAll('.event');
+  const modal = document.getElementById('event-details-modal');
+  const modalContent = modal.querySelector('.modal-content');
+
+  events.forEach(event => {
+    event.addEventListener('click', () => {
+
+      const eventAttributes = event.dataset;
+
+      modalContent.innerHTML = `
+        <div class="modal-flex-container">
+            <div class="modal-column">
```

Comment:
i don't like the html in javascript way
i think it would be way better to render each full event as it should be shown in the end, but all of them are hidden
then when you expand it simply make the correct one visible